src/app/contents/fields/generic.js

```diff
@@ -282,7 +278,7 @@ const fields = async () => {
       title: "platforms",
       label: "Platforms",
       description:
-        "This key specifies which platform the software runs on. It is meant to describe the platforms that users will use to access and operate the software, rather than the platform the software itself runs on. Use the predefined values if possible. If the software runs on a platform for which a predefined value is not available, a different value can be used. Values: web, windows, mac, linux, ios, android. Human readable values outside this list are allowed.",
```

Comment:
> the platforms that users will use to access and operate the software, rather than the platform the software itself runs on

I find the clarification confusing. Is it even needed?
Is it trying to say that we are not talking about the platform running the backend, but the user facing application? Wouldn't the backend application be in another repo anyway?

Reply:
@sebbalex WDYT?